src/example-types/numberUtil.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
           filter: rangeFilter(field, min, max),
           aggs: {
             statistical: {
-              stats: {
+              extended_stats: {
```

Comment:
Should this change be `0.9.0` instead of `0.8.4`? I mean, would this change the results?

Reply:
I don't think so. The result changes but isn't a breaking change because the new object still contains all the properties the previous object had.

Reply:
Perfect! Thank you!